go.mod

```diff
@@ -29,6 +30,8 @@ require (
 
 replace github.com/pion/interceptor v0.1.12 => github.com/streamer45/interceptor v0.0.0-20230202152215-57f3ac9e7696
 
+replace github.com/mattermost/mattermost-plugin-calls/server/public => ./server/public
```

Comment:
I actually tried to use Go workspaces to avoid this but ended up in a bit of rabbit hole with some unexpected failures on dependencies. Will look into this at another time as it's working fine for now.